scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -122,6 +127,7 @@ package object dialects {
     def allowOrTypes = Scala210.allowOrTypes
     def allowTraitParameters = Scala210.allowTraitParameters
     def allowLiteralTypes: Boolean = Scala210.allowLiteralTypes
+    def allowTrailingComma: Boolean = Scala210.allowLiteralTypes
```

Comment:
So this dialect is what we'll use in sbt 1, which will use Scala 2.12.2+ and therefore _will_ allow trailing commas. But in sbt between 0.13.7 and 1.0.0 (`[0.13.7,1.0.0)`) it's not. Can we model _that_ here?